### ISSUE TITLE
Hard to follow

### ISSUE THREAD

**Issue by mmmency** [GRHun/SBoxDetection#1]
Hard to follow this repo. A simple readme with experiment guide would help a lot.

**Comment by GRHun** [comment]
Thank you for your feedback! I just wrote a simple README file. Maybe it lacks a presentation of the graphical interface and detailed usage instructions.  Don't worry, I'll include that info in the next update to make it easier for everyone to use the tool. Thanks again for your suggestion, and stay tuned for the new changes☕!
